src/components/JoinLanding.tsx

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+
+const JoinLanding = () => {
+  return (
+    <div className="flex flex-col items-center justify-center bg-black p-10">
+      <div className="mb-5 h-16 w-4/5 rounded-lg bg-blue-500/60">
+        <div className="text-wrap px-10 py-3 text-center text-sm text-white">
+          We look forward to seeing you soon! We have practice every Monday and
+          Wednesday 8-10PM!
+        </div>
+      </div>
+
+      <div className="h-44 w-4/5 rounded-lg bg-gray-400"></div>
```

Comment:
Try to avoid hardcoding the h-44 if possible?
```suggestion
      <div className="h-44 w-4/5 rounded-lg bg-gray-400"></div>
```